meetings/2018-11-17.md

```diff
@@ -67,7 +67,9 @@
 
 **17:00-17:15 茶歇**
 
-**17:15-18:00 兴趣组的未来规划和工作模式**
+**17:15-17:35 翻译贡献与管理**
+
+**17:35-18:00 兴趣组的未来规划和工作模式**
```

Comment:
Should this be an internal discussion?

Reply:
I'm hoping to collect some feedback from the members and developers on how we can better serve for the community in the f2f meeting :)